packages/cdk/lib/roles/policies/nextflow-submit-job-batch-policy.ts

```diff
@@ -0,0 +1,24 @@
+import * as iam from "monocdk/aws-iam";
+
+export interface NextflowSubmitJobBatchPolicyProps {
+  account: string;
+  region: string;
+  submitJobPolicyArns: string[];
+}
+
+export class NextflowSubmitJobBatchPolicy extends iam.PolicyDocument {
+  constructor(props: NextflowSubmitJobBatchPolicyProps) {
+    const nextflowJobArn = `arn:aws:batch:${props.region}:${props.account}:job-definition/nf-*:*`;
```

Comment:
This is a little concerning that we have to build up the arn in here that way. Is it the best we could do?